ipv6cat/templates/ipv6cat/base.html

```diff
@@ -0,0 +1,101 @@
+{% load staticfiles bootstrap3%}
+{% load i18n %}
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
I think we can simplify this template; not borrowing any styles/js etc. from hosting ?